docs/source/markdown/podman-create.1.md

```diff
@@ -755,6 +755,9 @@ Security Options
 - `seccomp=unconfined` : Turn off seccomp confinement for the container
 - `seccomp=profile.json` :  White listed syscalls seccomp Json file to be used as a seccomp filter
 
+- `proc-opts=OPTIONS` : Comma separated list of options to use for the /proc mount.  More details for the
+  possible mount options are specified at **proc(5)** man page.
```

Comment:
proc(5) is not specified in the "See Also" section of this page and probably should be.

Reply:
fixed and pushed a new version